transport/transport-runtime/src/main/java/com/google/android/datatransport/runtime/logging/Logging.java

```diff
@@ -24,30 +24,51 @@ private static String getTag(String tag) {
   }
 
   public static void d(String tag, String message) {
-    Log.d(getTag(tag), message);
+    tag = getTag(tag);
+    if (Log.isLoggable(tag, Log.DEBUG)) {
+      Log.d(tag, message);
+    }
   }
 
   public static void d(String tag, String message, Object arg1) {
```

Comment:
Is this redundant of `public static void d(String tag, String message, Object... args) `.
Same applies to `public static void d(String tag, String message, Object arg1, Object arg2)`

Reply:
it's not redundant as it avoids an Object[] array allocation, it's common to overload like this for efficiency in the most common logging cases

Reply:
Gotcha!